extension/intents/tabs/tabs.toml

```diff
@@ -205,7 +205,7 @@ phrase = "Reset zoom"
 [tabs.previous]
 description = "Go to a previously active tab"
 match = """
-  (Go | switch | change | activate |) (the | my |) previous tab
+  (Go | switch | change | activate |) (to |) (the | my |) previous tab
```

Comment:
That would match "go to my previous tab" or "switch to my previous tab" etc. So I think we'd want to keep it.

Reply:
I have updated it @ianb.

Reply:
or may I close this PR because this change was not required?